assets/js/custom/Program.js

```diff
@@ -19,6 +19,17 @@ export const Program = function (projectId, projectName, userRole, myProgram, st
     })
   }
 
+  // -------------------------- Steal Button
+  $('.js-btn-project-steal').on('click', (e) => {
```

Comment:
if possible, try to prevent using jquery in your future contributions. It's unnecessary overhead we are trying to prevent.